packages/newsletter-sdk/index.tsx

```diff
@@ -66,27 +67,3 @@ export function useSubscribeToNewsletter(): [Subscription, SubscribeFn] {
 
    return [state, subscribe];
 }
-
-/**
- * Subscribe an email to the newsletter
- * @param email The email to subscribe
- */
-export async function subscribeToNewsletter(
-   apiOrigin: string,
-   email: string
-): Promise<void> {
-   const response = await sentryFetch(
-      `${apiOrigin}/api/2.0/cart/newsletter/subscribe`,
-      {
-         method: 'POST',
-         body: JSON.stringify({
-            email,
-         }),
-      }
-   );
-   if (response.status >= 300) {
```

Comment:
I looked at the source and this api always returns 200, so I'm fine that we dropped this code-specific error handling.